facade/service.go

```diff
@@ -1141,12 +1156,6 @@ func (f *Facade) scheduleOneService(ctx datastore.Context, tenantID string, svc
 		svc.DesiredState = int(desiredState)
 	}
 
-	err := f.validateServiceStart(ctx, svc)
```

Comment:
This shouldn't have been here; it was only called previously if desiredState != service.SVCStop